setup.py

```diff
@@ -46,9 +46,11 @@ def read_version(file_path="version.txt"):
     CUDAExtension,
     BuildExtension,
     CUDA_HOME,
+    ROCM_HOME,
```

Comment:
more of a nit but perhaps we should rename this PR to suporting custom HIP kernels

Reply:
thanks, will find a better name.